src/elements/flip-card/flip-card/readme.md

```diff
@@ -0,0 +1,42 @@
+The `sbb-flip-card` component displays an informative card that reveals more informations when clicked or toggled programmatically.
+It's meant to be used together with `sbb-flip-card-summary` and `sbb-flip-card-details`.
+
+```html
+<sbb-flip-card>
+  <sbb-flip-card-summary>
+    <sbb-title> Card Title </sbb-title>
+    <sbb-image slot="image" image-src="..."></sbb-image>
+  </sbb-flip-card-summary>
+  <sbb-flip-card-details> Some additional text. </sbb-flip-card-details>
+</sbb-flip-card>
+```
+
+## Slots
+
+The component will display the content slotted in the `summary` slot in the main view, and the content slotted inside the `details` slot after the card has been flipped.
+
+## States
+
+The `sbb-flip-card` will switch to the flipped state after the user clicks on it or after the `toggle` method is called. Once the component is flipped, it will have the attribute `data-flipped`, which is for internal use only, so it should not be set by hand.
```

Comment:
Not sure if the 'data-flipped' is really necessary for consumers..

Reply:
we had some similar 'data-stuff' logic elsewhere.. not documented at all => not intended for use 😃